packages/rulesets/src/spectral/test/system-data-in-properties-bag.test.ts

```diff
@@ -0,0 +1,62 @@
+import { Spectral } from "@stoplight/spectral-core"
+import linterForRule from "./utils"
+
+let nonResolvingLinter: Spectral
+const RULE = "SystemDataInPropertiesBag"
+const ERROR_MESSAGE = "System Data must be defined as a top-level property, not in the properties bag."
+
+beforeAll(async () => {
+  nonResolvingLinter = await linterForRule(RULE, true)
+  return nonResolvingLinter
+})
+
+test(`${RULE} should find errors`, async () => {
```

Comment:
>should [](http://example.com/codeflow?start=14&length=6)

Lets add another test for completeness where systemData is defined in a model referenced from properties.properties instead of inlining it